File: lambda/show_secrets.ts
import {
  GetSecretValueCommand,
  SecretsManagerClient,
} from "@aws-sdk/client-secrets-manager";

export const handler = async () => {
  const client = new SecretsManagerClient();

  try {
    const command = new GetSecretValueCommand({
      SecretId: process.env.SECRETS_NAME,
    });
    const response = await client.send(command);
    console.log(response);
    return {
      statusCode: 200,
      body: JSON.stringify(response),
    };
  } catch (error) {
    console.log(error);
    return {
      statusCode: 500,
      body: JSON.stringify("Error retrieving secret"),
    };
  }
};

File: lambda/python/show_secrets.py
import json
import os

import boto3


def lambda_handler(event, context):
    client = boto3.client("secretsmanager")
    response = client.get_secret_value(SecretId=os.environ["SECRETS_NAME"])
    print(response)
    secret = json.loads(response["SecretString"])
    return {
        "message": f"{secret["secret1"]}, {secret["secret2"]}, {secret["secret3"]}",
    }
